### PR TITLE
Dev event form

### PR DESCRIPTION
**Pull request by AlexisReliefApps**
Event page reorganizated

### CODE REVIEW

**Review by bamoure — APPROVED** [review]

File: src/components/HRInfoAddress.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+import React          from 'react';
```
Comment (guillaumev) [inline src/components/HRInfoAddress.js]: @AlexisReliefApps Why did you add this file instead of modifying the existing Address.js file ?

File: src/components/HRInfoAddress.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  },
+  withoutLabel: {
+    marginTop: theme.spacing.unit * 3,
+  },
+  textField: {
+    flexBasis: 200,
+  },
+});
+
+class HRInfoFiles extends React.Component {
```
Comment (guillaumev) [inline src/components/HRInfoAddress.js]: Wrong class name...

File: src/containers/DocumentForm.js [hunk 3]
```diff
@@ -1,334 +1,339 @@
-import React from 'react';
```
Comment (guillaumev) [inline src/containers/DocumentForm.js:1]: Please remove the modifications to DocumentForm.js from this PR, as it's not related...

File: src/containers/DocumentsPage.js [hunk 4]
```diff
@@ -67,15 +67,15 @@ class DocumentsPage extends React.Component {
     return (
```
Comment (guillaumev) [inline src/containers/DocumentsPage.js:67]: Please remove these modifications from this PR

File: src/containers/newEventForm.js [hunk 5]
```diff
@@ -0,0 +1,363 @@
+import React      from 'react';
```
Comment (guillaumev) [inline src/containers/newEventForm.js]: Feel free to put these modifications directly in EventForm.js

File: src/utils/Routes.js [hunk 6]
```diff
@@ -1,83 +1,92 @@
 import React from "react";
```
Comment (guillaumev) [inline src/utils/Routes.js:1]: Please remove these modifications from this PR

File: src/components/Address.js [hunk 7]
```diff
@@ -6,8 +6,8 @@ class Address extends React.Component {
   constructor(props) {
```
Comment (guillaumev) [inline src/components/Address.js]: Please remove these modifications from this PR

File: src/components/HRInfoLocation.js [hunk 8]
```diff
@@ -18,7 +18,7 @@ class HRInfoLocation extends React.Component {
   getOptions () {
```
Comment (guillaumev) [inline src/components/HRInfoLocation.js:18]: Please remove these modifications from this PR
